sdk/python/kfp/components/_python_op.py

```diff
@@ -38,6 +39,9 @@
 T = TypeVar('T')
 
 
+_FEATURE_STRIP_FILE_IO_NAME_PARTS = True
```

Comment:
nit: Maybe a one line comment documenting its functions?

Also, is there any case that we don't want strip out the suffix?

Reply:
>is there any case that we don't want strip out the suffix?

There should not be. I've removed the switch.